arcade-services.sln

```diff
@@ -115,7 +115,9 @@ Project("{2150E333-8FDC-42A3-9474-1A3956D46DE8}") = "ProductConstructionService"
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "ProductConstructionService.Api.Tests", "test\ProductConstructionService.Api.Tests\ProductConstructionService.Api.Tests.csproj", "{BB8CC065-79D6-48CE-AD96-32EDDDC4D404}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Maestro.Authentication", "src\Maestro\Maestro.Authentication\Maestro.Authentication.csproj", "{70D48B7A-DBFE-4762-A83F-4617ECF80838}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Maestro.Authentication", "src\Maestro\Maestro.Authentication\Maestro.Authentication.csproj", "{70D48B7A-DBFE-4762-A83F-4617ECF80838}"
```

Comment:
Why did this change? should we revert it?

Reply:
I think that's just something VS did